### PR TITLE
Fix corner cases for `case` and `catch` patterns

### PR DESCRIPTION
**Pull request by rvanasa**
Fixes a minor regression from #165.

### CODE REVIEW

File: tests/formatter.test.ts [hunk 1]
```diff
@@ -650,5 +650,7 @@ public type T = {
         await expectFormatted('_[i]\n');
         expect(await format('_ (i)')).toEqual('_(i)\n');
         expect(await format('_ [i]')).toEqual('_[i]\n');
+        expect(await format('case _ (i)')).toEqual('case _ (i)\n');
```
Comment (ggreif) [inline tests/formatter.test.ts:653]: Maybe also check `catch _ ()`...

**Review by ggreif — APPROVED** [review]
Nice! LGTM
